CLAUDE.md

```
@@ -191,6 +191,358 @@ All state lives in `src/renderer/stores/`. Key stores:
+
+### How to Recover Tasks
+
+**For JSON Errors (empty/malformed `implementation_plan.json`):**
+
+```bash
+# Main project
+cd "/path/to/project/.auto-claude/specs"
+for task in 071-marko 079-alpine-htmx-knockout 080-svelte-aurelia; do
+  cat > "$task/implementation_plan.json" << 'EOF'
+{
+  "feature": "Auto-recovery task",
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)